app/.env.local.sample

```diff
@@ -1,6 +1,9 @@
 # Environment variables for firebase-admin
 GOOGLE_CREDS_PATH=required, path to your service account json file that you downloaded from firebase console.
 
+# Environment variables encryption
+EVENT_USER_PASSPHRASE = "your super secret phrase"
+
```

Comment:
Not required since we will not use the `crypto-js`. Please remove it as well, and refer to the following review, for more details.

Thank you!